test/features/lb/private_lb.feature

```diff
@@ -33,6 +34,6 @@ Feature: Test a basic tfvars configuration for the lb module with a private tier
         Then terraform resource 'aws_lb' 'default' has changed attributes
             | attr     | value |
             #----------|-------|
-            | internal | true  |
+            | internal | ${bool:true}  |
```

Comment:
What is the difference between `true` and `${bool:true}`?

Reply:
`true` results in a string with the value of "true".
`${bool:true}` results in a true boolean datatype.

We could approach the issue with 2 different solutions:
1. It was simple enough to add boolean types to the interpolation process, and I figured this was desirable functionality anyway
2. We could instead change the code so that every value in the returned json-ish dict object is "stringified" and match values that way.

I think solution 1 was the more simple approach.

Reply:
I should mention that the tests fail when trying to compare a "true" string with a `true` boolean.